packages/grafana-ui/src/components/Combobox/filter.ts

```diff
@@ -0,0 +1,23 @@
+import { ComboboxOption } from './Combobox';
+
+export function itemToString<T extends string | number>(item?: ComboboxOption<T> | null) {
+  if (!item) {
+    return '';
+  }
+  if (item.label?.includes('Custom value: ')) {
```

Comment:
as this is just moved consider it non blocking, but would be nice to have this as a string constant so it's clear where the `Custom value:` string in an option is created so nobody accidentally change it without changing it also here.

Reply:
Now that we have support for descriptions, I think we could remove the need for this in the label altogether.